uxarray/grid/grid.py

```diff
@@ -347,6 +348,28 @@ def validate(self):
         else:
             raise RuntimeError("Mesh validation failed.")
 
+    def compute_face_center(self, method="average"):
```

Comment:
This may be more appropriate: 

```suggestion
    def construct_face_centers(self, method="average"):
```
or the following

```suggestion
    def construct_face_latlon(self, method="average"):
```

Any thoughts?

Reply:
yes, cartesian is better